README.md

```diff
@@ -79,7 +79,7 @@ plugins {
 
 </details>
 
-If you are not using convention plugins and would like a quick way of testing the plugin you can alternatively place it in the root project's build.gradle (change '3.0.1' to the latest version of the cache fix plugin
+If you are not using convention plugins and would like a quick way of testing the plugin you can alternatively place it in the root project's build.gradle (change '3.0.2' to the latest version of the cache fix plugin
```

Comment:
Since we are keeping these version numbers up to date, it seems redundant to say 'change '3.0.2' to the latest version of the ....'

Reply:
done